src/sentry/static/sentry/app/plugins/components/pluginIcon.tsx

```diff
@@ -5,6 +5,7 @@ import placeholder from 'app/../images/integrations/integration-default.png';
 import sentry from 'app/../images/integrations/integration-sentry.png';
 import asana from 'app/../images/integrations/asana-logo.png';
 import bitbucket from 'app/../images/integrations/bitbucket-logo.png';
+import bitbucketserver from 'app/../images/integrations/bitbucket-server-logo.png';
```

Comment:
@pierredup Can you resize the image to be 72x72? I realize that we aren't following this standard right now but going forward, that is going to be the requirement.